pkg/util/initsystem/initsystem.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2016 The Kubernetes Authors.
+Copyright 2019 The Kubernetes Authors.
```

Comment:
I don't think we change the year.

Reply:
@freeformz
@BenTheElder is correct and the project does not bump the date
we need to keep it 2016.

Reply:
@neolit123 this is now fixed.